modular_skyrat/modules/food_replicator/replicator.dm

```diff
@@ -0,0 +1,20 @@
+#define RND_CATEGORY_NRI_FOOD "Provision"
+#define RND_CATEGORY_NRI_MEDICAL "Medicine"
+#define RND_CATEGORY_NRI_CLOTHING "Apparel"
+
+/obj/machinery/biogenerator/food_replicator
+	name = "\improper BEEP colonial supplier"
+	desc = "This handy, ancient machine replicates food and clothing and medicine previously utilised by the pioneer colonists of what's now known as the NRI."
```

Comment:
desc = "The Type 34 'Colonial Supply Core,' colloquially known as the 'Gencrate/CSC' is an ancient, boxy design first put in use by the pioneer colonists of what's now known as the NRI. The Gencrate is at its core a matter resequencer, a highly specialized subtype of autolathe which performs a sort of transmutation using organic compounds; normally from large-scale crops or waste product. With sufficient supply, the machine is capable of making a wide variety of provisions, from clothes to food to potable water."

Reply:
Will be included in the next commit.

Reply:
please put lengthy descriptions in examine more please please.........

Reply:
It's, literally, three sentences; your brain won't melt.